tests/api/middleware/errorHandler.test.ts

```diff
@@ -0,0 +1,89 @@
+import { ResourceNotFoundError } from "utils/errors/resourceNotFoundError";
+import { ValidationError } from "utils/errors/validationError";
+import { BadError } from "utils/errors/badError";
+import { StatusCode } from "utils/statusCode";
+import { errorHandler } from "api/middlewares";
+
+describe("errorHandler", () => {

```

Comment:
Aqui só por padronização seria interessante começar com Given

```suggestion
describe("Given errorHandler", () => {

```